tools/scripts/use-node-version

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env bash
+
+# Check whether the current node version matches the .nvmrc version.
```

Comment:
it's somewhat outside the scope of this PR, but i don' think we can cater for that scenario. i'm not sure if that would present a real problem thoguh?

Reply:
I was wondering if a newer version of node is used to install packages for projects that specify an older version of node whether that could cause problems due to breaking API changes that might happen between versions. I'm only raising this because I've sometimes seen projects break when using a newer version of node to install deps (this was back when the platform was a lot younger to be fair).

I suppose another consideration might be that — if/when we move other projects to the monorepo — developers might want to develop using the version of node that their project targets when deployed to production.

You're right though, this is outside the scope of this PR and is probably a discussion to have elsewhere